FusionIIIT/applications/placement_cell/views.py

```diff
@@ -159,11 +160,12 @@ def get_reference_list(request):
         context = {
             'reference_objs': reference_objects
         }
-        return JsonResponse(context)
+    return HttpResponse(context)

```

Comment:
You have just defined an empty context object above which is rendered finally as a blank page, and the if condition is not even executing.